### DISCUSSION TITLE
Is the Spell Check supposed to work?

### DISCUSSION THREAD

**Discussion by SARTrack** [jendrikseipp/rednotebook#691] [Q&A]
When enabling the Spell Checker, every word is red. That is no wonder, because nowehere is an option to set the language.
How is this supposed to work?

**Comment by SARTrack** [comment]
And this is in the Windows version. There is only one tab in the Preferences, "General". Is there supposed to be more tabs maybe?
Also: Where are the settings stored? 
Where is the data stored?

**Comment by jendrikseipp** [comment]
Yes, it's supposed to work. Are you using the latest version? 2.29.6? Can you post the logfile from C:\Users\.rednotebook\rednotebook.log?

**Comment by lesdelc** [comment]
I am using RedNotebook version 2.30 for Windows 11 and did download Aspell (Aspell-en-0.50-2-3.exe) and installed it. After selecting Spell Check, Aspell underlines ALL text in RED instead of the misspelled word. How can I resolve this problem?
